### PR TITLE
Small bug fixes

### LINKED ISSUE #2713: Slight grammar error in event title
Further is misspelled as futher in the event's title.

image

### LINKED ISSUE #2723: SpCW: Rep.'s Burgos garrison decision incorrect effect
This mod is missing an older bugfix from vanilla.

The remove_effect section of SPA_expand_influence_in_the_burgos_garrison used to be bugged, and didn't have the intended effect when going down the Popular Front path. 

Fixed section, starting at line 1626 in /common/decisions/SPR.txt:

```
remove_effect = {
			if = {
				limit = {
					has_completed_focus = SPA_a_great_spain
				}
				176 = {
					custom_effect_tooltip = SPA_expand_influence_nationalists_tt
					set_state_flag = { flag = SPR_recently_expanded_influence_flag value = 1 days = 15 }
					hidden_effect = {
						if = {
							limit = { check_variable = { SPA_garrison_control < 4 } }
							
							add_to_variable = { var = SPA_garrison_control value = 1 }
							SPR = { add_to_variable = { var = spa_civil_war_divisions_counter_var value = 1 } }
							SPR = { subtract_from_variable = { var = spr_civil_war_divisions_counter_var value = 1 } }
						}
					}
					SPA_garrison_control_tooltip = yes
					SPA_garrison_control_cost_calculation = yes
				}
			}
			if = {
				limit = {
					has_completed_focus = SPR_the_popular_front
				}
				176 = {
					custom_effect_tooltip = SPR_expand_influence_republicans_tt
					set_state_flag = { flag = SPR_recently_expanded_influence_flag value = 1 days = 15 }
					hidden_effect = {
						subtract_from_variable = { var = SPA_garrison_control value = 1 }
						SPR = { subtract_from_variable = { var = spa_civil_war_divisions_counter_var value = 1 } }
						SPR = { add_to_variable = { var = spr_civil_war_divisions_counter_var value = 1 } }
					}
					SPA_garrison_control_tooltip = yes
					SPR_garrison_control_cost_calculation = yes
				}
			}
			clr_country_flag = SPA_already_expanding_influence_flag
} 
```

### PR DESCRIPTION
**Pull request by marijn211**
-Fixed a couple incorrect references to the Suez instead of Istria
-Made Italy use BBA DLC alt-history AI paths
-Restored Naples cores for fragmentation gamerules
-Fixed a typo
-Fixed a very minor error log entry
-German civil war now punishes deleting army
-Fixed setting Persia to go communist and Andorra to go flexible foreign policy
-Swapped which gamerule non-BBA players can select for Italy to go democratic, as this localization made much more sense for the non-BBA tree than the other democratic path gamerule loc that was used.
-Fixed Burgos influence boosting not working for the Spanish republic.
-Created previously missing AI plans for non-NSB users.
-Turkey can no longer defile *and* continue Ataturks legacy.

### CODE REVIEW

File: common/decisions/formable_nation_decisions.txt
Comment (ilkober) [inline common/decisions/formable_nation_decisions.txt:1]: if you remove the state from here, you should probably also remove the entry from decision category highlighting the required states

**Review by ilkober — COMMENTED** [review]
Looks good, waiting to response to my previous comment, otherwise seems fine for merging (looked over the code, wasn't tested in-game though)

File: common/decisions/formable_nation_decisions.txt
Comment (marijn211) [reply common/decisions/formable_nation_decisions.txt:1]: Its a duplicate entry, its not dupe in the highlighting

**Review by ilkober — APPROVED** [review]
Looks good

**Pull request merged**